nilearn/plotting/img_plotting.py

```diff
@@ -24,12 +24,12 @@
 import numpy as np
 from matplotlib import gridspec as mgs
 from nibabel.spatialimages import SpatialImage
-from nilearn._utils import _compare_version
```

Comment:
This is no longer needed ?

Reply:
flake8 was flagging them as unnecessary because unused so I removed them thinking "for sure this will break something and this will require a 'noqa' to pacify flake8"

so I am a bit puzzled nothing broke

will try to track down when they became unneeded.